sqflite/lib/src/sentry_batch.dart

```diff
@@ -68,6 +68,7 @@ class SentryBatch implements Batch {
     });
   }
 
+  /// @nodoc
```

Comment:
does it exist something similar to https://github.com/getsentry/sentry-javascript/blob/8482c0a02960d191d369db7af712ec48b9033f44/packages/core/src/hub.ts#L96 ?